spec/abci/abci++_comet_expected_behavior.md

```diff
@@ -118,7 +118,10 @@ Let us now examine the grammar line by line, providing further details.
   to provide the Application with all the snapshots needed, in order to reconstruct the state locally.
   A successful attempt must provide at least one chunk via `ApplySnapshotChunk`.
   At the end of a successful attempt, CometBFT calls `Info` to make sure the reconstructed state's
-  _AppHash_ matches the one in the block header at the corresponding height.
+  _AppHash_ matches the one in the block header at the corresponding height. Note that the state 
+  of  the application does not contain vote extensions itself. The application can rely on 
+  [CometBFT to ensure](./../../docs/rfc/rfc-100-abci-vote-extension-propag.md#base-implementation-persist-and-propagate-extended-commit-history)
+  the node has all the relevant data by the time it switches to consensus. 
```

Comment:
```suggestion
  the node has all the relevant data to proceed with the execution beyond this point. 
```
I'm skeptical about using "switching to consensus" in this section, as, from the point of view of the grammar (i.e., the App) consensus is not distinguishable from blocksync.

Reply:
Ah I reread the sentence and you comment. You are not talking about removing the switching to consensus section, just about rewording this. I agree ,we can change this.